src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp/Extensions/OmniSharpRazorCodeDocumentExtensions.cs

```diff
@@ -5,7 +5,7 @@
 using Microsoft.CodeAnalysis.Razor.Workspaces.Extensions;
 using Microsoft.CodeAnalysis.Text;
 
-namespace Microsoft.AspNetCore.Razor.OmniSharpPlugin;
+namespace Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Extensions;
 
 public static class OmniSharpRazorCodeDocumentExtensions
```

Comment:
Shoudl this be internal? etc. etc.

I won't comment on them all, but I don't know what this PR is exactly trying to achieve, so I don't know if this is right or wrong.